libvcx/src/aries/handlers/issuance/holder/state_machine.rs

```diff
@@ -42,14 +42,14 @@ impl HolderSM {
         self.source_id.clone()
     }
 
-    pub fn state(&self) -> u32 {
+    pub fn get_state(&self) -> HolderState {
         match self.state {
-            HolderState::OfferReceived(_) => VcxStateType::VcxStateRequestReceived as u32,
-            HolderState::RequestSent(_) => VcxStateType::VcxStateOfferSent as u32,
-            HolderState::Finished(ref status) => {
+            HolderFullState::OfferReceived(_) => HolderState::OfferReceived,
+            HolderFullState::RequestSent(_) => HolderState::RequestSent,
+            HolderFullState::Finished(ref status) => {
                 match status.status {
-                    Status::Success => VcxStateType::VcxStateAccepted as u32,
-                    _ => VcxStateType::VcxStateNone as u32,
+                    Status::Success => HolderState::Finished,
+                    _ => HolderState::Failed
```

Comment:
Please include numeric state changes in PR description (like here, `VcxStateNone` was none, now we have `HolderState::Failed` which is `3`

Reply:
I am not sure, are these important? Any user should be using the constants / enums. The numeric values of these are subject to change anyway.